client/src/app/components/managing-body/managing-body.component.ts

```diff
@@ -0,0 +1,21 @@
+import {Component, Input, Output, EventEmitter} from '@angular/core';
+
+@Component({
+  selector: 'op-managing-body',
+  templateUrl: 'app/components/managing-body/managing-body.component.html',
+  styleUrls: ['app/components/managing-body/managing-body.component.css'],
```

Comment:
@tarieli1 
It seems that the _css_ file is empty, so please leave this array empty and remove the _css_ file as well, thanks.